pyvista/themes.py

```diff
@@ -2989,7 +2986,24 @@ def lighting_params(self, config: _LightingConfig):
         self._lighting_params = config
 
 
-class DarkTheme(DefaultTheme):
+class DefaultTheme(Theme):
+    """Deprecated default theme.
+
+    .. deprecated:: 0.40.0
+        Deprecated and renamed to ``Theme``. No longer the default.
+
+    """
+
+    def __init__(self):
+        """Initialize the theme."""
+        super().__init__()
+        warnings.warn(
+            '`DefaultTheme` has been deprecated and renamed `Theme`. Further, it is no longer the default theme.',
```

Comment:
My suggestion after sitting on it... I think listing the new default theme helps point users in the right direction in case they want to follow the new recommended default rather than stay on the old default.

```suggestion
            '`DefaultTheme` has been deprecated and renamed `Theme`. Further, `DocumentTheme` is now the PyVista default theme'.,
```